.azure_pipelines/olive-ci.yaml

```diff
@@ -85,20 +85,14 @@ jobs:
     name: Linux_CI
     pool: $(OLIVE_POOL_UBUNTU2004)
     onnxruntime: onnxruntime
+    subfolder: local
```

Comment:
nit: subfolder is not required anymore. not an issue, just think this keeps it consistent with the stable and nightly yamls

Reply:
Removed.